Src/RE2.Native/RE2.Native.x64.vcxproj

```diff
@@ -61,11 +61,6 @@
   <PropertyGroup Label="UserMacros" />
   <!-- Compilation Settings and Source Files are all defined here, in a shared props file -->
   <Import Project="RE2.Native.props" />
-  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
```

Comment:
>ItemDefinitionGroup [](start = 3, length = 19)

Why did you delete these?  Was it intentional? These are important link time optimizations.

Reply:
Oh, I see, you've moved the configuration to the props file. :)

---
In reply to: [571650449](https://github.com/microsoft/sarif-pattern-matcher/pull/174#discussion_r571650449) [](ancestors = 571650449)